src/components/ErrorBoundary/ErrorBoundaryWithMessage.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Alert } from '@patternfly/react-core';
+import ErrorBoundary from './ErrorBoundary';
+
+interface MessageProps {
+  message: string;
+}
+
+export default class ErrorBoundaryWithMessage extends React.Component<MessageProps> {
+  alert() {
+    return (
+      <div className="card-pf-body">
+        <Alert variant="warning" title={this.props.message || 'Something went wrong rending this component'}>
```

Comment:
This style wont' be in PF4, maybe you can change it or add the CardBody component of PF4

Reply:
I changed that for bit. I used the Card and CardBody components :)